kubernetes/create-config-maps.sh.j2

```diff
@@ -35,4 +38,6 @@ kubectl create configmap concent-api-settings                                \
 
 kubectl create configmap gatekeeper-settings                               \
     --from-file=local_settings.py=config-maps/gatekeeper/local_settings.py \
+    --from-file=admin_settings.py=config-maps/concent/admin_settings.py    \
+    --from-file=email_settings.py=config-maps/concent/email_settings.py    \
```

Comment:
I think we should add `timing_settings` to gatekeeper too for consistency.

Do it in a separate commit of course - because it's not just refactoring but a change in behavior.

Reply:
Done.